### ISSUE TITLE
The extended attribute and smart groups don't seem to be working.

### ISSUE THREAD

**Issue by rseidensf** [kc9wwh/MakeMeAdminPy#5]
I added the EA by copying and pasting the script from here, then I created the 2 EAs per the instructions, and I also created 2 smart groups also according to instructions but it doesn't seem to be working correctly. 

If a Mac is in the admin state, shouldn't that Mac appear in the "MakeMeAdmin - Violations" smart group and if is no longer in that state, shouldn't that Mac be in the "MakeMeAdmin - Remediated"one? If so, this isn't working correctly. I tested 2 Macs and they both ran the "MakeMeAdmin" policies successfully, but after checking in and reporting inventory, they remain in the "Violations" smart group, even though they are no longer in the admin state.

Please advise when you can. 

Much appreciated,
Rich

**Comment by kc9wwh** [comment]
Hi @rseidensf - 

So there are 3 options that the EA can present...

**Compliant** - _This occurs when everything works correctly and the user didn't attempt to create another admin account and/or change the orgAdmin credentials._

**Remediated** - _This will happen if the user creates a new admin account or they change the password on the orgAdmin account, but we were able to successfully remove the admin rights from the newly created admin account and reset the password back to what is expected for the orgAdmin._

**Violation** - _This will only happen if the user changes the orgAdmin password and for some reason we are unable to reset it._

Either way...**Remediated** and **Violation** are both bad users. This flag will "reset" back to **Compliant** whenever they run the policy and complete the workflow successfully without any malice intent. I did also just test out the workflow using the EA and everything appears to be working.

Let me know if you have any further questions.
